wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/TaskDefinitionElementToWomTaskDefinition.scala

```diff
@@ -57,8 +57,20 @@ object TaskDefinitionElementToWomTaskDefinition {
         }.map(_.toSeq)
       }
 
-      (validRuntimeAttributes, validCommand) mapN { (runtime, command) =>
-        CallableTaskDefinition(a.taskDefinitionElement.name, Function.const(command.validNel), runtime, Map.empty, Map.empty, taskGraph.outputs, taskGraph.inputs, Set.empty, Map.empty)
+      val metaSection : ErrorOr[Map[String, String]] = a.taskDefinitionElement.metaSection match {
+        case None => Map.empty.validNel
```

Comment:
Although it compiles just fine, IntelliJ red-squiggles this claiming it should be a `Map.empty[String, String].validNel`.